src/main/kotlin/racing/domain/common/CarRaceProperty.kt

```diff
@@ -0,0 +1,12 @@
+package racing.domain.common
+
+import racing.domain.car.Car
+
+class CarRaceProperty(val roundSize: Int) {
+    lateinit var cars: List<Car>
+        private set
+
+    constructor(roundSize: Int, carNames: List<String>) : this(roundSize) {
+        cars = carNames.map { Car(it) }
```

Comment:
```suggestion
        cars = carNames.map(::Car)
```
이런 경우에는 다음처럼 function reference를 활용할 수 있어요! lambda식에 비해 전달 인자를 조작하지 않고 원본 그대로 넘김을 보장할 수 있다는 이점이 있습니다. function reference과 lambda를 decompile했을때 내부에서 어떻게 동작하는지 알고 싶으시면 다음을 참고해보세요.
https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845

Reply:
오 알지 못했던 부분입니다. 공유 감사합니다-!